SU-SWS/sulgryphon-nextjs
# SUL23-456: adding an svg title

# READY FOR REVIEW
- Added a `title` to the SVG icons

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Checkout the test build link.
2. Look at the icons on the pages listed below and see they have an associated title.
3. https://su-library-git-sul23-456-svg-title-stanford-libraries.vercel.app/people/benjamin-albritton - Link icon on right
4. https://su-library-git-sul23-456-svg-title-stanford-libraries.vercel.app/spaces/places-study
5. https://su-library-git-sul23-456-svg-title-stanford-libraries.vercel.app/libraries-0/branches-and-centers 


# Affected Projects or Products
- No

# Associated Issues and/or People
[- SUL23-456](https://stanfordits.atlassian.net/browse/SUL23-456)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
